source/common/config/grpc_mux_impl.cc

```diff
@@ -8,14 +8,17 @@
 namespace Envoy {
 namespace Config {
 
-GrpcMuxImpl::GrpcMuxImpl(const envoy::api::v2::Node& node, Grpc::AsyncClientPtr async_client,
+GrpcMuxImpl::GrpcMuxImpl(const envoy::api::v2::core::Node& node, Grpc::AsyncClientPtr async_client,
                          Event::Dispatcher& dispatcher,
                          const Protobuf::MethodDescriptor& service_method)
     : node_(node), async_client_(std::move(async_client)), service_method_(service_method) {
   retry_timer_ = dispatcher.createTimer([this]() -> void { establishNewStream(); });
 }
 
 GrpcMuxImpl::~GrpcMuxImpl() {
+  // Hack to force linking of the service: https://github.com/google/protobuf/issues/4221
```

Comment:
Sorry, can you add a TODO here to fix this so we can search for it? Also please add a TODO to my new test which I didn't do. Otherwise LGTM.

Reply:
added todo.